recipe/build.sh

```diff
@@ -32,17 +28,20 @@ EOF
 
 LINKFLAGS="${LINKFLAGS} -L${LIBRARY_PATH}"
 
-CXX=${CXX_FOR_BUILD:-${CXX}} CC=${CC_FOR_BUILD:-${CC}} ./bootstrap.sh \
+CXXFLAGS="$(echo ${CXXFLAGS} | sed 's/ -march=[^ ]*//g' | sed 's/ -mcpu=[^ ]*//g' |sed 's/ -mtune=[^ ]*//g')" \
+CFLAGS="$(echo ${CFLAGS} | sed 's/ -march=[^ ]*//g' | sed 's/ -mcpu=[^ ]*//g' |sed 's/ -mtune=[^ ]*//g')" \
+    CXX=${CXX_FOR_BUILD:-${CXX}} CC=${CC_FOR_BUILD:-${CC}} ./bootstrap.sh \
     --prefix="${PREFIX}" \
     --with-toolset=${TOOLSET} \
     --with-icu="${PREFIX}" \
     --with-python="${PYTHON}" \
-    --with-python-root="${PREFIX} : ${PREFIX}/include/python${PY_VER}m : ${PREFIX}/include/python${PY_VER}" \
-    2>&1
+    --with-python-root="${PREFIX} : ${PREFIX}/include/python${PY_VER}" \
+    || (cat bootstrap.log; exit 1)
```

Comment:
This comes from [boost-cpp](https://github.com/conda-forge/boost-cpp-feedstock/blob/f6b34b392599fe6aa4ef48a679a90d289e5f646e/recipe/build.sh#L38).

Clarity is clearly pretty subjective (I'm a noob in bash, but your suggestions aren't particularly clearer to me[^1]), but I think the intent is pretty clearly: "in case of failure, print log, _then_ fail". Would the following be a less ambiguous expression of that?
```suggestion
    || (cat bootstrap.log && exit 1)
```

[^1]: I wouldn't have guessed that `false` fails the build, and wouldn't know how `{ }` is different from `( )` here.